### PR TITLE
Make the event documentation valid according to the remark linter

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-tracking bundle size difference from 2.0.10 to 485f648ce0a56d58abad09347ec42bbfe2b5c37d
js: 4.59kb increase (1.37kb/gzip)

File: docs/event.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 This allows a low barrier to entry for anyone wanting to submit events to Spoor.
 
-__Important__: To decide how to name the category and action fields, consider this sentence (square brackets denote that part is optional):
+__Important__: To decide how to name the category and action fields, consider this sentence (angle brackets denote that part is optional):
 
-*A user can {action} a {category}[ with/having a {context}[ to {destination}]]*
+*A user can {action} a {category}< with/having a {context}< to {destination}>>*
```
Comment (chee) [inline docs/event.md:11]: does it allow using `\[` to mean a literal squacket?

Comment (chee) [reply docs/event.md:11]: \[like this\](https://i-am-not-a-link.com)

Comment (JakeChampion) [reply docs/event.md:11]: That does work :-)

Comment (JakeChampion) [reply docs/event.md]: I've done that in https://github.com/Financial-Times/o-tracking/pull/289/commits/7f3fce7633ce04710f9b2e499831711546b1ea31 now :-)

**Comment by origamiserviceuser** [comment]
o-tracking bundle size difference from 2.0.10 to 45ac45563b29f1973facd3a23a2b5b7d4e5c0aec
js: 4.59kb increase (1.37kb/gzip)

**Comment by chee** [comment]
very nice!!

**Pull request merged**